orka_vector_api/views/jobs.py

```diff
@@ -198,13 +206,19 @@ def put_job(job_id):
         job = get_job_by_id(job_id, conn, current_app)
         if job is None:
             current_app.logger.info(f'Could not update job {job_id}. Job not found.')
-            raise Exception("Job not found.")
+            raise OrkaException("Job not found.")
         update_job(job_id, conn, current_app, **post_body)
         db.pool.putconn(conn)
-    except Exception:
+        response = json.dumps({'success': True}), 201, {'ContentType': 'application/json'}
+    except OrkaException:
+        response = json.dumps({'success': False}), 400, {'ContentType': 'application/json'}
```

Comment:
```suggestion
        response = json.dumps({'success': False}), 404, {'ContentType': 'application/json'}
```
technically we would need to differentiate which code this should be. Maybe a second parameter to OrkaException?